AdminWebsite/AdminWebsite/Security/UserRole.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace AdminWebsite.Security
+{
+    public enum UserRole
+    {
+        [Display(Name = "None")] None = 0,
+        [Display(Name = "VH Officer")] VhOfficer = 1,
+        [Display(Name = "Representative")] Representative = 2,
+        [Display(Name = "Individual")] Individual = 3,
+        [Display(Name = "Judge")] Judge = 4,
+        [Display(Name = "Case Admin")] CaseAdmin = 5
```

Comment:
This is just me not knowing, what do we use the display name for?

Reply:
Don't use it.. removed!